metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Mouse Follows Focus",
     "description": "Are you a power-user?\nDo you like using Super+1,2,3 to access your favorite apps?\nAre you annoyed that you have to manually move your mouse between screens because it can't keep up with your keyboard shortcuts?\nThen this extension is for you!\n\nThis simple GNOME shell extension does the opposite of the 'focus follows mouse' setting. It makes the mouse follow your keyboard focus. Whenever you focus a window, if the cursor isn't already in it, it will jump to the windows center, making it easy to interact with it.",
     "uuid": "mousefollowsfocus@matthes.biz",
-    "shell-version": ["45"],
+    "shell-version": ["45", "46"],
```

Comment:
You can add also `47` to supported list. I already did it in my fork and I'm running it daily on Fedora 41 and it works like a charm.

I'm leaving this commit link here just for posterity: https://github.com/DrOptix/mousefollowsfocus/commit/dfffbc8de9f5c3cd4f05c358e24a567fb4ed6a42